core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -393,11 +388,16 @@ class TransactionStateManager(brokerId: Int,
       loadingPartitions.add(partitionAndLeaderEpoch)
     }
 
-    def loadTransactions(): Unit = {
+    def loadTransactions(startTimeInMs: java.lang.Long = 0L): Unit = {
```

Comment:
no need to have a default value for the parameter.

Reply:
Agree, done.